patches/server/0094-LootTable-API-Replenishable-Lootables-Feature.patch

```diff
@@ -603,7 +603,25 @@ index e2a9782e0ae60eb5557dce0831084c5722687df6..9c15af9b622716ba8b21b8b1138ad1cd
              ContainerHelper.loadAllItems(nbt, this.getItemStacks());
          }
  
-@@ -184,4 +184,13 @@ public interface ContainerEntity extends Container, MenuProvider {
+@@ -104,13 +104,15 @@ public interface ContainerEntity extends Container, MenuProvider {
+ 
+     default void unpackChestVehicleLootTable(@Nullable Player player) {
+         MinecraftServer minecraftServer = this.getLevel().getServer();
+-        if (this.getLootTable() != null && minecraftServer != null) {
++        if (this.getLootableData().shouldReplenish(player) && this.getLootTable() != null && minecraftServer != null) { // Paper
```

Comment:
``this.getLootTable() != null`` check is not needed,
shouldReplenish does a null check already.